libs/api/domains/social-insurance/src/lib/dtos/temporaryCalculation.input.ts

```diff
@@ -0,0 +1,70 @@
+import { InputType, Field, Int } from '@nestjs/graphql'
+
+@InputType('SocialInsuranceIncomeType')
+class IncomeType {
+  @Field(() => Int, { nullable: true })
+  incomeTypeNumber?: number
+
+  @Field()
+  incomeTypeCode?: string
+
+  @Field()
+  incomeTypeName?: string
+
+  @Field()
+  currencyCode?: string
+
+  @Field(() => Int, { nullable: true })
+  incomeCategoryNumber?: number
+
+  @Field()
+  incomeCategoryCode?: string
+
+  @Field()
+  incomeCategoryName?: string
+
+  @Field(() => Number, { nullable: true })
```

Comment:
NestJS would rather we either use Int og Float for the number decorator
![image](https://github.com/user-attachments/assets/ef1b15b2-ca38-46c3-9a16-6a1cd9b858ed)